sbt-scalajs-bundler/src/main/scala/scalajsbundler/sbtplugin/ScalaJSBundlerPlugin.scala

```diff
@@ -460,6 +461,9 @@ object ScalaJSBundlerPlugin extends AutoPlugin {
 
     scalaJSModuleKind := ModuleKind.CommonJSModule,
 
+    // Ask Scala.js to run the main module on script load
+    scalaJSUseMainModuleInitializer := true,
```

Comment:
You should not force this on the users. They should include that setting themselves.